app/models.py

```diff
@@ -134,6 +134,12 @@ class User(db.Model, ModelMixin, UserMixin):
         db.Boolean, nullable=False, default=False, server_default="0"
     )
 
+    # Fields for WebAuthn
+    fido_uuid = db.Column(db.String(), nullable=True, unique=True)
+    fido_credential_id = db.Column(db.String(), nullable=True, unique=True)
+    fido_pk = db.Column(db.String(), nullable=True, unique=True)
+    fido_sign_count = db.Column(db.Integer(), nullable=True)
```

Comment:
Do you know how `fido_sign_count` would be used?

Reply:
I think it's to prevent the replay attack.
https://github.com/duo-labs/py_webauthn/blob/18c837594059bdcca354e3154231ebd468d74b41/webauthn/webauthn.py#L1089